pandas/core/base.py

```diff
@@ -942,7 +942,8 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
 
         See Also
         --------
-        numpy.ndarray.argmax
```

Comment:
@datapythonista is this a convention we're moving to?

Reply:
Hmm yea can be pretty repetitive /tedious but I don’t think there’s a way around it with the current docstring validation unless @datapythonista knows otherwise

Reply:
In this case my preference would be to leverage the context of the page we are in, and have as a description something like "Same function for numpy arrays.". 

But I personally think that it adds value to have the elements in the see also, and a short comment on why they are related. I'm thinking on a new user visiting `head`, and how helpful it can be to know that a `tail` method exists, and that it "does the same as head but from the end of the dataframe".

Would you prefer to not have the descriptions?